satellite/attribution/db.go

```diff
@@ -23,10 +23,23 @@ type Info struct {
 	CreatedAt  time.Time
 }
 
+// ValueAttributionRow represents data from QueryValueAttribution without exposing dbx
+type ValueAttributionRow struct {
```

Comment:
I'm not sure if we should use everywhere `ValueAttribution`. In metainfo we have `SetAttribution` so maybe we can skip `Value` in metod/struct names. In this particular case we have `attribution.ValueAttributionRow` and maybe `attribution.CSVRow` will be more descriptive (similar to payments).

Reply:
Updated